### DISCUSSION TITLE
odd and even variants does not work (Please Help)

### DISCUSSION THREAD

**Discussion by harinathpamu** [tailwindlabs/tailwindcss#3148] [Help] [answered]
![Html](https://user-images.githubusercontent.com/24853247/102712951-c2a05500-42ea-11eb-9227-d65ffae15ffd.png)
![Config](https://user-images.githubusercontent.com/24853247/102712955-c7fd9f80-42ea-11eb-9183-cb359d287175.png)

It selects everything : (

Please Help ...

**Comment by adamwathan** [comment, marked as answer]
Hey! You need to use those utilities directly on the element you are trying to affect, not the parent:

https://play.tailwindcss.com/mdE23PLBQI

```html
<div>
  <div class="odd:bg-blue-500">Row</div>
  <div class="odd:bg-blue-500">Row</div>
  <div class="odd:bg-blue-500">Row</div>
  <div class="odd:bg-blue-500">Row</div>
  <div class="odd:bg-blue-500">Row</div>
  <div class="odd:bg-blue-500">Row</div>
</div>
```

They are designed to be used in loops.

**Reply by harinathpamu** [reply]
Thank you very much : )
Your awesome guy thanks once again
